content/03-reference/01-tools-and-interfaces/01-prisma-schema/06-relations.mdx

```diff
@@ -6,28 +6,35 @@ metaDescription: 'A relation is a connection between two models in the Prisma sc
 
 ## Overview
 
-This is an extension of the [Data model](data-model) page which describes how to handle relations between Prisma models in the Prisma schema.
-
 > **Note**: If you are upgrading from a Prisma 2.0 Preview version (e.g. `preview025`) and want to adopt the new relation syntax, be sure to check out the [release notes](https://github.com/prisma/prisma/releases) which contain concrete instructions for the upgrade process.
 
-A relation is a _connection_ between two models in the Prisma schema. A relation must always be defined with two _relation fields_, one on each the model. In the case of one-to-one and one-to-many relations, an additional _relation scalar field_ must be added:
+A relation is a _connection_ between two models in the Prisma schema. For example, there is a one-to-many relation between `User` and `Post` because one user can have many blog posts. The following Prisma schema defines a one-to-many relation between `User` and `Post`. The fields involved in defining the relation are highlighted:
 
-```prisma
+```prisma line-number
 model User {
   id      Int      @id @default(autoincrement())
-  posts   Post[]
+|  posts   Post[]
 }
 
 model Post {
   id         Int         @id @default(autoincrement())
-  author     User        @relation(fields: [authorId], references: [id])
-  authorId   Int         // relation scalar field  (used in the `@relation` attribute above)
+|  author     User        @relation(fields: [authorId], references: [id])
+|  authorId   Int         // relation scalar field  (used in the `@relation` attribute above)
 }
 ```
 
-> **Note**: In SQL, relations between models are represented via _foreign keys_ in the underlying database. A foreign key is only stored on _one_ side of a relation. As opposed to how relations are modeled in SQL, **Prisma always requires explicit relation fields on _both_ sides of the relation to be set on your Prisma models**.
->
-> This means that one side of the relation represents the _foreign key_ in the underlying database, the other relation field is provided on a Prisma-level (in the case of implicit many-to-many relations, both relation fields are provided on a Prisma-level).
+At a Prisma level, a relation is made up of:
+
+- Two [_relation fields_](#relation-fields): `author` and `posts`. Relation fields are used to define connections between models at the Prisma level and **do not exist in the database**.
+- The scalar `authorId` field. This field **does** exist in the database and represents the foreign key that connect a `Post` record to a `User` record.
```

Comment:
The second item here makes me stumble: 
Although it is true that the relation fields to not really exist, but the relation scalar field does, the "foreign key" part (which is a separate thing for me in SQL) is _not_ represented in the scalar field. This is indeed in the main relation field (which includes its target).

Reply:
Hmmmm..I'm not sure I follow you. `authorId` is the foreign key in the database. Is the word 'scalar' in the sentence 'scalar `author` field' the part that doesn't work?

Reply:
Let me rephrase:
`authorId` is a column in the database, that also has some FK stuff on it.
In our schema we have a `authorId` field, but the FK stuff in represented in the relation field with `@relation`.